mailbox/api/src/test/java/org/apache/james/mailbox/events/EventBusContract.java

```diff
@@ -97,17 +128,48 @@ default void dispatchShouldNotThrowWhenAGroupListenerFails() {
             .doesNotThrowAnyException();
     }
 
+    @Test
+    default void dispatchShouldKeepWorkingIfGroupListenerFails() {
```

Comment:
Given the live explanation given, I propose **failingGroupListenersShouldNotAbortGroupDelivery**